src/proto/Ethereum.proto

```diff
@@ -3,6 +3,32 @@ syntax = "proto3";
 package TW.Ethereum.Proto;
 option java_package = "wallet.core.jni.proto";
 
+message TransferContract {
+    // Amount to send in wei (256-bit number)
+    bytes amount = 1;
```

Comment:
transfer could contain `data`

Reply:
Really?  Then what's the difference between the geenric smart contract call an transfer?

Reply:
OK, optional payload added